### PR TITLE
rudiuk

### PR DESCRIPTION
**Pull request by oleksandr-rudiuk**

### CODE REVIEW

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
You should push the readme file and add the demo links to the description according to instructions

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,151 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="POST"></form>
```
Comment (DarkMistyRoom) [inline src/index.html:14]: all inputs should be placed inside the form

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,151 @@
...
+    <fieldset class="personalinfo">
+      <legend>
+        Personal information
+      </legend> 
+      <div class="first">
```
Comment (DarkMistyRoom) [inline src/index.html:19]: Add empty lines between multiline sibling blocks of HTML

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,151 @@
...
+          type="time" 
+          name="time"
+        >
+      </div>
+      <div class="formfield">
+        What are your favorite brand of cars?
+        <select name="auto" multiple>  
+          <option value="BMW">
+          BMW
+          </option>
```
Comment (DarkMistyRoom) [inline src/index.html:118]:
```suggestion
          <option value="BMW">
            BMW
          </option>
```

Comment (DarkMistyRoom) [reply src/index.html:118]: fix it for the other options

File: src/index.html [hunk 4]
```diff
@@ -11,7 +11,151 @@
...
+            Yes
+          </option>
+          <option value="no">
+            No
+          </option>
+        </select>  
+      </div>
+    </fieldset>
+    <button 
+      type="submit">
```
Comment (DarkMistyRoom) [inline src/index.html:155]:
```suggestion
    <button type="submit">
```

File: src/style.css [hunk 5]
```diff
@@ -1 +1,8 @@
-/* styles go here */
+.personalinfo,
+.registration {
```
Comment (DarkMistyRoom) [inline src/style.css:2]: you can do it with only one class

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 6]
```diff
@@ -11,7 +11,151 @@
...
+          placeholder="email@example.com"
+        >
+      </div>
+      <div class="formfield">
+        Password:
+        <input type="password" name="password" 
+          required
+          minlength="5"
+          maxlength="15"
+        >
```
Comment (ViktorHavryliuk) [inline src/index.html:77]: See checklist p.3
```suggestion
        <input 
          type="password" 
          name="password" 
          required
          minlength="5"
          maxlength="15"
        >
```

**Review by ViktorHavryliuk — CHANGES_REQUESTED** [review]
Labels are missing:
image
Also, feedback of previous mentor seems not to be implemented. Please check if you have pushed correct changes

**Pull request closed without merging**